jwt/utils.py

```diff
@@ -1,17 +1,19 @@
 import base64
 import binascii
 import struct
+from typing import Union
 
 try:
+    from cryptography.hazmat.primitives.asymmetric.ec import EllipticCurve
     from cryptography.hazmat.primitives.asymmetric.utils import (
         decode_dss_signature,
         encode_dss_signature,
     )
 except ImportError:
-    pass
+    EllipticCurve = object  # type: ignore
```

Comment:
`Any` is probably a better choice here.
`Any` basically means dynamic typing, i.e. we don't know if it will work or not.
`object` basically means we don't care what type is passed, it is always type safe.
If you defined this without the import above, so Mypy used `object`, you would find there are typing errors in those functions (object has not attribute key_size).

Alternatively, you could even define it with:
```
class EllipticCurve(Protocol):
    @property
    def key_size(self) -> int: ...
```

Probably not worth it though, given that these functions will fail anyway if the import failed.

Reply:
Thanks switched to `Any`.

mypy runs with cryptograpy installed, so the real class will always exist during mypy runs. IMO, we should use the upstream definition as much as possible rather than duplicating it.